.gitignore

```diff
@@ -8,6 +8,7 @@
 *.pem
 *.crt
 *.env
+*.exe
```

Comment:
this will only work on Windows, not linux (no extension)

We should find a way to exclude the output folder.